aws-ts-eks/cluster.ts

```diff
@@ -0,0 +1,457 @@
+import * as pulumi from "@pulumi/pulumi";
+import * as aws from "@pulumi/aws";
+import * as k8s from "@pulumi/kubernetes";
+import * as k8sInputs from "@pulumi/kubernetes/types/input";
+
+import { Dashboard } from "./dashboard";
+import { ServiceRole } from "./serviceRole";
+
+/**
+ * EBSVolumeType lists the set of volume types accepted by an EKS storage class.
+ */
+export type EBSVolumeType = "io1" | "gp2" | "sc1" | "st1";
```

Comment:
Not a big deal - but would be nice if this was defined in `@pulumi/aws`.

Reply:
I agree. The various components used in this example really ought to live in their own package, frankly.

Reply:
Yep. I wasn't suggesting that we avoid adding this ASAP.